go.mod

```diff
@@ -1,8 +1,15 @@
 module github.com/gophercloud/gophercloud
 
+go 1.17
+
 require (
 	github.com/mitchellh/mapstructure v1.5.0
 	golang.org/x/crypto v0.0.0-20190211182817-74369b46fc67
-	golang.org/x/sys v0.0.0-20190209173611-3b5209105503 // indirect
 	gopkg.in/yaml.v2 v2.2.2
 )
+
+require (
+	github.com/alecthomas/units v0.0.0-20211218093645-b94a6e3cc137 // indirect
```

Comment:
I'm not a fan of referencing new packages, can you let me know why are you requiring new packages? Otherwise, please remove that